### PR TITLE
Removed 2 lines from  main

### PR DESCRIPTION
**Pull request by Illyra**

### CODE REVIEW

File: TanDang_FinalProject/actor-actress.csv [hunk 1]
```diff
@@ -1401,15 +1401,15 @@ Year,Award,Winner,Name,Film
...
+2002,Actress in a Leading Role,0,Ren��e Zellweger,Chicago
 2002,Actress in a Supporting Role,0,Kathy Bates,About Schmidt
 2002,Actress in a Supporting Role,0,Julianne Moore,The Hours
 2002,Actress in a Supporting Role,0,Queen Latifah,Chicago
 2002,Actress in a Supporting Role,0,Meryl Streep,Adaptation
 2003,Actor in a Leading Role,1,Sean Penn,Mystic River
 2003,Actor in a Supporting Role,1,Tim Robbins,Mystic River
 2003,Actress in a Leading Role,1,Charlize Theron,Monster
-2003,Actress in a Supporting Role,1,Ren�e Zellweger,Cold Mountain
+2003,Actress in a Supporting Role,1,Ren��e Zellweger,Cold Mountain
```
Comment (henrytan2) [inline TanDang_FinalProject/actor-actress.csv]: take a look at this

File: TanDang_FinalProject/main.cpp [hunk 2]
```diff
@@ -1,7 +1,5 @@
 #include <iostream>
 
 int main() {
-    std::cout << "Hello World!" << std::endl;
-    std::cout << "asdf" << std::endl;
```
Comment (henrytan2) [inline TanDang_FinalProject/main.cpp]: nice

File: TanDang_FinalProject/actor-actress.csv (hunk 1 above)
Comment (Illyra) [reply TanDang_FinalProject/actor-actress.csv]: fixed

File: TanDang_FinalProject/main.cpp (hunk 2 above)
Comment (Illyra) [reply TanDang_FinalProject/main.cpp]: drgfd

**Pull request merged**